src/data-interfaces/HoloFuelDnaInterface.js

```diff
@@ -249,7 +248,7 @@ const HoloFuelDnaInterface = {
       const params = since ? { since } : {}
 
       const { transactions } = await createZomeCall('transactions/list_transactions')(params)
-      const nonActionableTransactions = transactions.map(presentTransaction)
+      const nonActionableTransactions = transactions.map(presentTransaction).filter(tx => !(tx instanceof Error))
```

Comment:
think you want to filter the list before mapping `presentTransaction` over it. `presentTransaction(anError)` is not defined

Reply:
obv, same feedback everywhere you're using this pattern

Reply:
@robbiecarlton, I dont think I understand. 
According to me the map build the array with the errors and then we filter it.

Reply:
My mistake. I misread it, you're right.